crypto/keys/ed25519/ed25519.go

```diff
@@ -0,0 +1,232 @@
+package ed25519
+
+import (
+	"crypto/subtle"
+	"fmt"
+	"io"
+
+	"github.com/tendermint/tendermint/crypto"
+	tmed25519 "github.com/tendermint/tendermint/crypto/ed25519"
+	"github.com/tendermint/tendermint/crypto/tmhash"
+	"golang.org/x/crypto/ed25519"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+)
+
+//-------------------------------------
```

Comment:
This key is only used in testing right?

Reply:
You're probably right, I'm not 100% sure. Maybe @blushi has a clearer idea?

In any case, it's preparing the groundwork for https://github.com/cosmos/cosmos-sdk/pull/7268#discussion_r492219055

Reply:
yeah afaik it's only used in tests for now